Assignment2/Elia-elia.alonsoru.ttl

```diff
@@ -0,0 +1,14 @@
+@base <http://www.example.org/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+:Class01	:includes	:Sensor 029 ;
+		:includes 	:Computer 101 .
```

Comment:
This is wrong. No blank space should be there between Computer and 101. Same with the sensor